crates/corro-agent/src/api/public/mod.rs

```diff
@@ -474,6 +481,13 @@ pub async fn api_v1_queries(
     let (data_tx, mut data_rx) = channel(512);
 
     tokio::spawn(async move {
+        if !pms.ndjson {
```

Comment:
I think we could do `Accept: application/ndjson` as a header. At least it's generally how content type is negotiated :)

I just invented that, but maybe there's a better mime type for that.